app/models/miq_server/status_management.rb

```diff
@@ -3,14 +3,39 @@
 module MiqServer::StatusManagement
   extend ActiveSupport::Concern
 
+  def status_update
+    self.assign_attributes(system_status)
+    self.assign_attributes(process_status)
```

Comment:
Changed to use assign_attributes instead of merging unrelated hashes.  🎩  🏑  ✨   cc @Fryguy @chrisarcand